content/blog/2020/12/2020-12-03-election-results.adoc

```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+title: "2020 Jenkins Board and Officer Elections Results"
+tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- oleg_nenashev
+- slide_o_mix
+- uhafner
+opengraph:
+  image: /images/governance/elections/2020/opengraph.png
+description: >
+  This blogpost summarizes
+---
+
+image:/images/governance/elections/2020/opengraph.png[Jenkins Elections, role=center, float=right, height=178]
+
+The Jenkins community has recently completed the 2020 elections.
+On behalf of the Jenkins community and the elections committee,
+we congratulate all newly elected board members and officers!
```

Comment:
Does this include the uncontested elections?

Reply:
Yes. Can reword it make it more explicit